src/solc_0.7/catalyst/CollectionCatalystMigrations.sol

```diff
@@ -79,6 +83,9 @@ contract CollectionCatalystMigrations is WithAdmin {
                 // solhint-disable-next-line no-empty-blocks
             } catch {}
         }
+        for (uint256 i = 0; i < gemIds.length; i++) {
```

Comment:
Why do we add 1 to the gems ?

Reply:
The contracts should expect to receive correct gemIds, not old one

Reply:
Ok, I see we use the old catalyst Id so only fair we use it for gems.
sounds good actually

Reply:
lets rename the argument to be oldCatalystId oldGemsIds to make it clearer

Reply:
ok :)